### PR TITLE
Fixups multiple dts + Documentation related stuff

### PR DESCRIPTION
**Pull request by okias**
affected:
- p4wifi
- chagall
- glide
- ms surface

### CODE REVIEW

File: arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi [hunk 1]
```diff
@@ -1051,7 +1051,7 @@
 
 		/* Azurewave AW-NH615 BCM4329B1 or AW-NH665 BCM4330B1 */
 		bluetooth {
-			compatible = "brcm,bcm4329-bt", "brcm,bcm4330-bt";
+			compatible = "brcm,bcm4329-bt";
```
Comment (digetx) [inline arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi:1054]: TF201 has bcm4329, TF300/700T has bcm4330. You should set "brcm,bcm4330-bt" in the TF300/700T dts then, or vice versa.

File: arch/arm/boot/dts/tegra30-pegatron-chagall.dts [hunk 2]
```diff
@@ -1699,6 +1699,7 @@
 
 	power-domain-core {
 		power-supply = <&vdd_core>;
+		#power-domain-cells = <0>;
```
Comment (digetx) [inline arch/arm/boot/dts/tegra30-pegatron-chagall.dts:1702]: The `power-domain-core` node should be removed now entirely. I missed to remove it from chagall and transformers-common dts.

File: arch/arm/boot/dts/tegra20-p4wifi.dts [hunk 3]
```diff
@@ -684,9 +688,8 @@
 
 	i2c-12 {
 		compatible = "i2c-gpio";
-		gpios = <
-			&gpio TEGRA_GPIO(O, 4) TEGRA_I2C_GPIO_FLAGS
-			&gpio TEGRA_GPIO(O, 2) TEGRA_I2C_GPIO_FLAGS >;
+		sda-gpios = <&gpio TEGRA_GPIO(O, 4) TEGRA_I2C_GPIO_FLAGS >;
+		scl-gpios = <&gpio TEGRA_GPIO(O, 2) TEGRA_I2C_GPIO_FLAGS >;
```
Comment (digetx) [inline arch/arm/boot/dts/tegra20-p4wifi.dts:692]: The whitespace after `TEGRA_I2C_GPIO_FLAGS` is unnecessary.

File: arch/arm/boot/dts/tegra30-pegatron-chagall.dts (hunk 2 above)
Comment (okias) [reply arch/arm/boot/dts/tegra30-pegatron-chagall.dts:1702]: ok, I'll change to patch to remove it

File: arch/arm/boot/dts/tegra20-p4wifi.dts (hunk 3 above)
Comment (okias) [reply arch/arm/boot/dts/tegra20-p4wifi.dts:692]: kk

File: arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi (hunk 1 above)
Comment (okias) [reply arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi:1054]: ok

Comment (okias) [reply arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi]: I dropped the patch for now, let the Transformers guys decide what they want.

File: arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi [hunk 4]
```diff
@@ -1050,8 +1050,7 @@
 					   <1000000 4000000 136>;
 
 		/* Azurewave AW-NH615 BCM4329B1 or AW-NH665 BCM4330B1 */
-		bluetooth {
-			compatible = "brcm,bcm4329-bt", "brcm,bcm4330-bt";
```
Comment (digetx) [inline arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi:1054]: This compatible shouldn't be removed.

Comment (digetx) [reply arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi:1054]: Or you should add it to to per-device dts.

Comment (okias) [reply arch/arm/boot/dts/tegra30-asus-transformer-common.dtsi:1054]: sorry, my bad, I thought I dropped this whole commit. let me fix it

File: arch/arm/boot/dts/tegra20-p4wifi.dts (hunk 3 above)
Comment (digetx) [inline arch/arm/boot/dts/tegra20-p4wifi.dts:692]: Not all whitespaces are removed, I'll fix it after applying.

**Comment by digetx** [comment]
Thanks!

**Pull request merged**